CHANGELOG.md

```diff
@@ -6,6 +6,22 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.2.0] - 2020-09-01
+
+### Added
+- Add `state` variable to Conjur Ansible role, which can be used to cleanup
+  configuration and identity artifacts created on managed nodes.
+  [cyberark/ansible-conjur-collection#176](https://github.com/cyberark/ansible-conjur-collection/pull/176)
+
+### Changed
+- Lookup plugin now retries variable retrieval 5 times before accepting a
+  failure response.
+  [cyberark/ansible-conjur-collection#60](https://github.com/cyberark/ansible-conjur-collection/pull/60)
+
+### Removed
+- End support for Python 2.
+  [cyberark/ansible-conjur-collection#69](https://github.com/cyberark/ansible-conjur-collection/pull/69)
```

Comment:
Do we need a new major release for this?

Reply:
All we did was drop testing against Python 2 - does that count as a backwards-incompatible change? There hasn't been substantial change to the Lookup plugin since then.

Regardless - do you think the project being Certified warrants a major release?

Reply:
I'm not sure. Maybe @jtuttle or @doodlesbykumbi can give their opinion?

Reply:
Minor seems fine to me.